src/msg.lisp

```diff
@@ -75,9 +75,26 @@
 (defmethod symbol-codes ((msg-type symbol))
   nil)
 
+(defmethod symbol-codes ((m ros-message))
+  (symbol-codes (type-of m)))
+
 (defmethod symbol-code ((m ros-message) s)
   (symbol-code (type-of m) s))
 
+(defmethod code-symbols ((msg-type symbol) code)
+  (remove code (symbol-codes msg-type) :test-not #'= :key #'rest))
+
+(defmethod code-symbols ((m ros-message) code)
+  (code-symbols (type-of m) code))
+
+(defmethod code-symbol ((msg-type symbol) code)
+  (let ((pair (rassoc code (symbol-codes msg-type) :test #'=)))
+    (when pair
```

Comment:
(It's not like I want it, I'm only sharing my thoughts and suggestions. I'll try to phrase my sentences better next time.)